### PR TITLE
Update readme to include getting head typescript

### PR DESCRIPTION
**Pull request by alexeagle**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+## Use latest TypeScript compiler
+TypeScript 1.5 beta will include everything you need. Until that is released,
+build your own TypeScript at HEAD. Follow the
+[instructions on the TypeScript repo](https://github.com/Microsoft/TypeScript/#building)
+
+## Start up the compiler
+Because we built our own TypeScript, the command to run it is longer:
+
+    $ cd ng2-ts-hello
+    $ node ../TypeScript/built/local/tsc.js -w
```
Comment (rkirov) [inline README.md:12]: don't you need --emitDecoratorMetadata?

Comment (rkirov) [inline README.md:12]: OOC, if you don't specify a file it would watch `**/*.ts`?

Comment (alexeagle) [inline README.md:12]: it reads tsconfig.json so you don't need any flags.
It does indeed compile a new file I add later, so I think these are redundant - will remove.

Comment (rkirov) [inline README.md:12]: Makes sense, didn't know about tsconfig.json, thought it is Atom-typescript specific.

Comment (alexeagle) [inline README.md:12]: Nope, that was Atom compiling the file I added later. It evaluates the glob when you start up tsc but doesn't see new files added later. So I think this is good to include main.ts here.
Also atom is helpfully (?) re-adding the files section when I delete it, by evaluating the glob:
https://github.com/TypeStrong/atom-typescript/blob/master/docs/tsconfig.md

I guess this is fine for the quickstart

**Pull request merged**